### PR TITLE
Feature/VTA-562 - Vehicle Lookup

### PR DESCRIPTION
**Pull request by FBarrassDVSA**
Vehicle lookup page
Search criteria modal
Multiple tech record selection page
Vehicle details page

### CODE REVIEW

File: src/app/pages/vehicle/vehicle-lookup/multiple-tech-records-selection/multiple-tech-records-selection.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        this.logProvider.dispatchLog({
+          type:
+            'error-vehicleService.getTestResultsHistory-openVehicleDetails in multiple-tech-records-selection.ts',
+          message: `${oid} - ${error.status} ${error.error} for API call to ${error.url}`,
+          timestamp: Date.now()
+        });
+
+        this.trackErrorOnRetrieval(ANALYTICS_VALUE.TEST_RESULT_HISTORY_FAILED);
+
+        this.storageService.update(STORAGE.TEST_HISTORY + selectedVehicle.systemNumber, []);
```
Comment (tomcrawleyy) [inline src/app/pages/vehicle/vehicle-lookup/multiple-tech-records-selection/multiple-tech-records-selection.ts:80]: is there some async stuff missing from this block?

**Comment by FBarrassDVSA** [comment]
Vehicle Lookup page 
image

**Comment by FBarrassDVSA** [comment]
Search Criteria modal
image

**Comment by FBarrassDVSA** [comment]
Multiple tech Record Selection page
image

**Pull request merged**